src/components/Dialog.vue

```diff
@@ -209,15 +212,16 @@ watch(
       let loading = ref(false)
       return {
         ...action,
-        loading,
+        loading: loading.value,
```

Comment:
This breaks reactivity IIRC. Even I had tried changing it to be mapped to value

Reply:
@ruchamahabal Thanks for pointing out, fixed.